recipes/gargammel/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
+sed -i.bak 's/make -C libgab/make -C libgab CXX=$(CC)/' Makefile
+sed -i.bak 's/make -C src/make -C src CXX=$(CC)/' Makefile
+sed -i.bak 's/CXXFLAGS =/CXXFLAGS +=/' src/Makefile
+sed -i.bak 's/LDFLAGS  =/LDFLAGS +=/' src/Makefile
+
+## Following https://bioconda.github.io/contributor/troubleshooting.html#g-or-gcc-not-found
+make CC=$CXX
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cp -R * $outdir/
```

Comment:
It appears that you are copying over all contents of the build directory to the $PREFIX (including Makefiles and object files etc.), but the $PREFIX should only contain the stuff that should be part of the package. This also explains the `patchelf` error messages (they refer to `.o` files).

Reply:
It appears that the binaries that the main `gargammel.pl` script needs are listed in line 236ff:
```
$adptsim = $pathdir."/src/adptSim";
my $deamsim = $pathdir."/src/deamSim";
my $fragsim = $pathdir."/src/fragSim";
my $artprog = $pathdir."/art_src_MountRainier/art_illumina";
```
So it may be sufficient to just copy those from `src/` to `$PREFIX/bin`.

Reply:
Ah, thank you for the explanation of cause of the `patchelf` messages. That makes sense, I modified the script after a more simpler recipe, where it didn't require so much complication.